tutoraspects/patches/k8s-deployments

```diff
@@ -30,9 +30,9 @@ spec:
           image: {{ DOCKER_IMAGE_CLICKHOUSE }}
           name: clickhouse
           ports:
-            - containerPort: 8123
-            - containerPort: 9000
-            - containerPort: 9009
+            - containerPort: "{{ CLICKHOUSE_HOST_INSECURE_HTTP_PORT }}"
+            - containerPort: "{{ CLICKHOUSE_HOST_INSECURE_NATIVE_PORT }}"
+            - containerPort: "{{ CLICKHOUSE_HOST_TLS_NATIVE_PORT }}"
```

Comment:
Note: the default values for these variables differ from what was hardcoded here, so operators may need to update their configurations.

The default values are:

* [`CLICKHOUSE_HOST_INSECURE_HTTP_PORT: 8123`](https://github.com/openedx/tutor-contrib-aspects/blob/3d6399109d73c4b87fafb1bd0d145f3407cf4971/tutoraspects/plugin.py#L144) (unchanged)
*  [`CLICKHOUSE_HOST_INSECURE_NATIVE_PORT: 9006`](https://github.com/openedx/tutor-contrib-aspects/blob/3d6399109d73c4b87fafb1bd0d145f3407cf4971/tutoraspects/plugin.py#L117) (changed from 9000)
* [`CLICKHOUSE_HOST_TLS_NATIVE_PORT: 9440`](https://github.com/openedx/tutor-contrib-aspects/blob/3d6399109d73c4b87fafb1bd0d145f3407cf4971/tutoraspects/plugin.py#L123) (changed from 9006)

Reply:
We are running Aspects with Clickhouse outside the cluster. I think it's safe to make the change